snuba/web/rpc/v1/legacy/trace_item_attribute_names.py

```diff
@@ -0,0 +1,107 @@
+import uuid
```

Comment:
nothing changes in this file from the eap_spans implementation. Once the double write is done, this file can be safely deleted